resources/assets/less/bem/beatmap-discussion-new.less

```diff
@@ -23,21 +23,18 @@
 
   &__footer {
     display: flex;
-    flex-wrap: wrap;
```

Comment:
The buttons look rather wrong. The box message could also use some padding (this one is an existing problem though).

![](https://s.myconan.net/2022-08/firefox_2022-08-29_14-22-03.png)

Reply:
More like text is too long and should be wrapping anyway.

I intentionally changed this so the timestamp message would not shift the buttons:
![image](https://user-images.githubusercontent.com/1694544/187135023-570e6ccd-775b-4701-8e4c-172398b16b3a.png)
![image](https://user-images.githubusercontent.com/1694544/187134926-90d74583-4be0-4eaf-baf0-5a1f0dc0b0a9.png)

Either the text should be wrapping or the elements should always be stacking (or buttons should not grow which then causes elements to not align visually)

Reply:
I don't think the tall button is a good compromise

Reply:
Would also prefer the buttons to not get taller.